clvm/more_ops.py

```diff
@@ -147,6 +149,16 @@ def op_divmod(args):
     return cost, args.to((q, r))
 
 
+def op_div(args):
+    cost = DIV_BASE_COST
+    i0, i1 = args_as_int_list("div", args, 2)
```

Comment:
This looks good. Is the performance much different than divmod?

I guess we need to wait for Rust benchmarking